src/progressbar/progressbar.js

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import * as React from 'react';
+import {getOverrides} from '../helpers/overrides';
+import {
+  Root as StyledRoot,
+  Bar as StyledBar,
+  Label as StyledLabel,
+} from './styled-components';
+
+import type {ProgressBarPropsT} from './types';
+
+class ProgressBar extends React.Component<ProgressBarPropsT, {}> {
```

Comment:
I think you can leave the state type as undefined? (remove the `, {}`)

Reply:
ok